index.html

```diff
@@ -29,59 +46,73 @@
     <div id="main">
         <!-- headline start -->
         <section id="headline">
```

Comment:
- [ ] For your header/headline section I noticed a few differences between your design and Figma :cry:   Please make sure that you stick to the design as much as possible (e.g., font, colors, images, text, margins) because it is very important to pay attention to details as a developer :+1: 

#### Figma design
![Screenshot from 2022-09-29 21-47-12](https://user-images.githubusercontent.com/6615394/193148507-e3bd11d7-de1d-4f4e-86fa-a0ce12863733.jpg)

#### Your design
![Screenshot from 2022-09-29 21-47-28](https://user-images.githubusercontent.com/6615394/193148509-d15241b6-e2d1-4403-bbe2-8b2171d51091.png)